lib/project_types/script/layers/infrastructure/errors.rb

```diff
@@ -146,6 +146,15 @@ class InvalidProjectError < ScriptProjectError; end
         class ScriptUploadError < ScriptProjectError; end
         class ProjectConfigNotFoundError < ScriptProjectError; end
         class InvalidProjectConfigError < ScriptProjectError; end
+
+        class ScriptTooLargeError < ScriptProjectError
+          attr_reader :max_size
+
+          def initialize(max_size)
+            super()
```

Comment:
Nit: I don't think you need the parentheses here.

Reply:
Ruby does this annoying thing where if you omit the parentheses in a `super` call, it passes the arguments through to the super method. While this will work in this case, I don't know if that makes sense since `max_size` isn't a message string.

Reply:
Yea probably fine to just keep the parentheses. I wasn't aware. Adam mentioned in our pairing session yesterday.